data/mapping.json

```diff
@@ -22300,6 +22300,18 @@
       "astID": "9337353612735049028",
       "sastID": "7386"
     },
+    {
+      "astID": "9629060213451111272",
+      "sastID": "7388"
```

Comment:
Is there any possibility of this colliding with a version older than 9.5.3?

Reply:
@constantinoantunes1 I think not. In this case, this query was only created in 9.5.3, it appears "before" existing entries from 9.5.2 probably because that ID ("7388") was reserved but not used before, from SAST side